.github/subproject_test/meson.build

```diff
@@ -0,0 +1,52 @@
+project('subpoj_tester', 'c')
+
+rz_deps = [
+  dependency('rz_core'),
+  dependency('rz_reg'),
+  dependency('rz_flag'),
+  dependency('rz_hash'),
+  dependency('rz_bin'),
+  dependency('rz_bp'),
+  dependency('rz_io'),
+  dependency('rz_search'),
+  dependency('rz_sign'),
+  dependency('rz_cons'),
+  dependency('rz_lang'),
+  dependency('rz_socket'),
+  dependency('rz_type'),
+  dependency('rz_debug'),
+  dependency('rz_il'),
+  dependency('rz_demangler'),
+  dependency('rz_util'),
+  dependency('rz_main'),
+  dependency('rz_crypto'),
+  dependency('rz_config'),
+  dependency('rz_egg'),
+  dependency('rz_syscall'),
+  dependency('rz_magic'),
+  ]
+
+# TODO: extend to perform actually linking test as well!
+tester_source = custom_target('main.c',
+  command: ['echo', '''
+#include <stdio.h>
```

Comment:
maybe add `rz_core.h` instead of `stdio.h` and call `rz_cons_printf`

Reply:
Can be easily done. However, note that any actual linker- or compiler-time-dependency has been avoided here intentionally as it will always be partial and I'm not sure if there is any guarantee by rizin to provide stability here (despite the stability of `rz_cons_printf` could be easily assumed)?